src/WebJobs.Script/Workers/WorkerConstants.cs

```diff
@@ -35,6 +35,17 @@ public static class WorkerConstants
         public const string LanguageWorkerConsoleLogPrefix = "LanguageWorkerConsoleLog";
         public const string FunctionConsoleLogCategoryName = "Host.Function.Console";
 
+        /// <summary>
+        /// The log category to be used for logging Tooling log entries emitted from language workers.
+        /// </summary>
+        public const string FunctionToolingConsoleJsonCategoryName = "Host.Function.ToolingConsoleJson";
```

Comment:
nit - Thinking it would be more consistent to add "Log" to this var name but that might make the name too long - I am wondering if "Function" in "FunctionConsoleLogCategoryName" is necessary (/what it indicates?) and if that can be removed from both vars so that you'd have "ToolingJsonConsoleLogCategoryName" and "ConsoleLogCategoryName" ?

Reply:
I wonder if json has to be in the name, I feel like `Host.Function.ToolingConsoleLog` is more appropriate. Will there be any other kind of tooling log what we would have to differentiate between json and not json?

Agree with Maddie to add Log to the name for consistency

Reply:
Good suggestions. Made the below changes in the latest iteration pushed.

- Renamed the logger category name to "Host.Function.ToolingConsoleLog"
- Renamed the constants to remove the "Functions" prefix.